Content.Server/Research/Components/RoboticsConsoleComponent.cs

```diff
@@ -0,0 +1,53 @@
+using Content.Server.Research.Systems;
+using Content.Shared.Radio;
+using Content.Shared.Research;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Server.Research.Components;
+
+[RegisterComponent, Access(typeof(RoboticsConsoleSystem))]
+public sealed partial class RoboticsConsoleComponent : Component
+{
```

Comment:
Shared on god.

Reply:
i started out having it in shared but then realised exactly none of it can be done in shared

Reply:
When I add UI window prediction I don't want robotics console to be the odd component out that doesn't predict.